### ISSUE TITLE
No mixin named mdc-ripple-common error after adding @material/react-menu/index.scss

### ISSUE THREAD

**Issue by ndtreviv** [material-components/material-components-web-react#901]
I recently added `@material/react-menu` to my project, and imported the scss file into mine amongst others as per the README, like so:
```
@import "@material/react-menu/index.scss";
@import "@material/react-top-app-bar/index.scss";
@import "@material/react-material-icon/index.scss";
```

However, I get the following error:
```
@include mdc-ripple-common($query);
          ^
      No mixin named mdc-ripple-common
      in /Users/someone/gitrepos/myproject/my-ui/node_modules/@material/menu/_mixins.scss (line 35, column 12)
```

If I comment out the react-menu import:
```
// @import "@material/react-menu/index.scss";
```
then it all works, and icons have a ripple effect when I click them.

Is there something else I need to do to get these configured properly?

Contents of .env:
```
SASS_PATH=node_modules
```

Result of `npm ls | grep material`:
```
├─┬ @material/react-material-icon@0.13.0
│ ├─┬ @material/react-ripple@0.13.0
│ │ ├─┬ @material/dom@1.1.0
│ │ ├─┬ @material/ripple@1.1.0
│ │ │ ├── @material/animation@1.0.0 deduped
│ │ │ ├── @material/base@1.0.0 deduped
│ │ │ ├── @material/dom@1.1.0 deduped
│ │ │ ├── @material/feature-targeting@0.44.1 deduped
│ │ │ ├── @material/theme@1.1.0 deduped
├─┬ @material/react-menu@0.13.0
│ ├─┬ @material/menu@1.1.1
│ │ ├─┬ @material/base@1.0.0
│ │ ├── @material/feature-targeting@0.44.1
│ │ ├─┬ @material/list@1.1.1
│ │ │ ├── @material/base@1.0.0 deduped
│ │ │ ├── @material/dom@1.1.0 deduped
│ │ │ ├── @material/feature-targeting@0.44.1 deduped
│ │ │ ├─┬ @material/ripple@1.1.0
│ │ │ │ ├── @material/animation@1.0.0 deduped
│ │ │ │ ├── @material/base@1.0.0 deduped
│ │ │ │ ├── @material/dom@1.1.0 deduped
│ │ │ │ ├── @material/feature-targeting@0.44.1 deduped
│ │ │ │ ├── @material/theme@1.1.0 deduped
│ │ │ ├── @material/rtl@0.42.0 deduped
│ │ │ ├── @material/shape@1.1.1 deduped
│ │ │ ├── @material/theme@1.1.0 deduped
│ │ │ ├── @material/typography@1.0.0 deduped
│ │ ├─┬ @material/menu-surface@1.1.1
│ │ │ ├── @material/animation@1.0.0 deduped
│ │ │ ├── @material/base@1.0.0 deduped
│ │ │ ├── @material/elevation@1.1.0 deduped
│ │ │ ├── @material/feature-targeting@0.44.1 deduped
│ │ │ ├── @material/rtl@0.42.0 deduped
│ │ │ ├── @material/shape@1.1.1 deduped
│ │ │ ├── @material/theme@1.1.0 deduped
│ │ ├─┬ @material/ripple@1.1.0
│ │ │ ├── @material/animation@1.0.0 deduped
│ │ │ ├── @material/base@1.0.0 deduped
│ │ │ ├── @material/dom@1.1.0 deduped
│ │ │ ├── @material/feature-targeting@0.44.1 deduped
│ │ │ ├── @material/theme@1.1.0 deduped
│ │ ├── @material/rtl@0.42.0
│ ├─┬ @material/react-list@0.13.0
│ │ ├── @material/dom@1.1.0 deduped
│ │ ├── @material/list@1.1.1 deduped
│ │ ├─┬ @material/react-checkbox@0.10.0
│ │ │ ├─┬ @material/checkbox@0.41.0
│ │ │ │ ├── @material/animation@0.41.0
│ │ │ │ ├── @material/base@0.41.0
│ │ │ │ ├─┬ @material/ripple@0.41.0
│ │ │ │ │ ├── @material/animation@0.41.0
│ │ │ │ │ ├── @material/base@0.41.0
│ │ │ │ │ └── @material/theme@0.41.0
│ │ │ │ ├── @material/rtl@0.40.1
│ │ │ │ ├─┬ @material/selection-control@0.41.0
│ │ │ │ │ └── @material/ripple@0.41.0 deduped
│ │ │ │ └── @material/theme@0.41.0
│ │ │ ├─┬ @material/react-ripple@0.10.0
│ │ │ │ ├── @material/ripple@0.41.0 deduped
│ │ ├─┬ @material/react-radio@0.10.0
│ │ │ ├─┬ @material/form-field@0.41.0
│ │ │ │ ├── @material/base@0.41.0
│ │ │ │ ├── @material/rtl@0.40.1
│ │ │ │ ├── @material/selection-control@0.41.0 deduped
│ │ │ │ ├── @material/theme@0.41.0
│ │ │ │ └── @material/typography@0.41.0
│ │ │ ├─┬ @material/radio@0.41.0
│ │ │ │ ├── @material/animation@0.41.0
│ │ │ │ ├── @material/base@0.41.0
│ │ │ │ ├── @material/ripple@0.41.0 deduped
│ │ │ │ ├── @material/selection-control@0.41.0 deduped
│ │ │ │ └── @material/theme@0.41.0
│ │ │ ├─┬ @material/react-ripple@0.10.0
│ │ │ │ ├── @material/ripple@0.41.0 deduped
│ │ ├── @material/react-ripple@0.13.0 deduped
│ ├─┬ @material/react-menu-surface@0.13.0
│ │ ├── @material/menu-surface@1.1.1 deduped
├─┬ @material/react-top-app-bar@0.13.0
│ ├── @material/react-ripple@0.13.0 deduped
│ ├─┬ @material/top-app-bar@1.1.1
│ │ ├─┬ @material/animation@1.0.0
│ │ ├── @material/base@1.0.0 deduped
│ │ ├─┬ @material/elevation@1.1.0
│ │ │ ├── @material/animation@1.0.0 deduped
│ │ │ ├── @material/feature-targeting@0.44.1 deduped
│ │ │ └── @material/theme@1.1.0 deduped
│ │ ├─┬ @material/ripple@1.1.0
│ │ │ ├── @material/animation@1.0.0 deduped
│ │ │ ├── @material/base@1.0.0 deduped
npm│ │ │ ├── @material/dom@1.1.0 deduped
│ │ │ ├── @material/feature-targeting@0.44.1 deduped
│ │ │ ├── @material/theme@1.1.0 deduped
│ │ ├── @material/rtl@0.42.0 deduped
│ │ ├─┬ @material/shape@1.1.1
│ │ │ └── @material/feature-targeting@0.44.1 deduped
│ │ ├─┬ @material/theme@1.1.0
│ │ │ └── @material/feature-targeting@0.44.1 deduped
│ │ ├─┬ @material/typography@1.0.0
│ │ │ └── @material/feature-targeting@0.44.1 deduped
```

Application was created by create-react-app and uses node-sass to watch and compile scss files.

**Comment by alangleontes** [comment]
try chaging your .env content to:

`SASS_PATH=.\node_modules`

**Comment by ndtreviv** [comment]
> `SASS_PATH=.\node_modules`

I'm on a Mac and have already tried:
> `SASS_PATH=./node_modules`

No joy :(

**Comment by ndtreviv** [comment]
The plot thickens: the problem goes away once I `yarn add @material/react-list`

**Comment by jonmanzo** [comment]
This is a dependency version mess - least in my case the best I can figure is this:

react-menu > react-list (0.13.0)

react-list deps:

`{
    "@material/dom": "^1.1.0",
    "@material/list": "^1.0.0",
    "@material/react-checkbox": "^0.10.0",
    "@material/react-radio": "^0.10.0",
    "@material/react-ripple": "^0.13.0",
    "classnames": "^2.2.6",
    "memoize-one": "^5.0.4",
    "react": "^16.4.2"
  }`

Where react-radio requires: `"@material/react-ripple": "^0.10.0"`

For whatever reasons (that I don't have time to figure out), react-radio/checkbox seem to be sticking their older dependencies (0.10.0 when the caret should get 0.13.0) at the root of the @material tree.

Thus, I'm getting 0.41.0 at @material/ripple and the required 1.1.0 at @material/react-ripple/node_modules/@material/ripple. Because the sass is being required directly with no dep chain, it finds the wrong version that has no mdc-ripple-common mixin...

I was able to solve by `yarn add @material/ripple` but I think the only thing that needs to really be done is bumping the radio/checkbox deps to ^0.13.0

**Comment by moog16** [comment]
@jonmanzo yes I also see that the @mateiral/ripple is resolving to 0.41.0. `checkbox` and `radio` should be updated to 0.13.0, which should resolve this issue.

**Comment by moog16** [comment]
released a patch 0.13.2
